src/SubmitABook/SubmitABook.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import bookService from '../firebase/bookService';
+
+const SubmitABook = function() {
+
+	function submitBook(e) {
+		e.preventDefault();
+		const formVals = {};
+		[...document.querySelectorAll('.input')].forEach(input => {
```

Comment:
What is the ... doing here, shouldn't document.querySelectorAll('.input') get you an array with all of the inputs on the page anyways?